<!-- Launch file to move on a preprogrammed trajectory -->

<launch>
	<group ns="ias"> <group ns="robot"> 
	
		<arg name="arm_name" default="arm_1"/>
	
		<group ns="$(arg arm_name)">  	
  	
			<node name="test_gesture_client" pkg="irob_autosurg" 
			type="test_gesture_client" output="screen">
  		
					<param name="arm_name" 
						type="string" value="$(arg arm_name)" />
						
					
		
	</node>
  	</group>
  	</group> </group>
</launch>
